src/Features/Core/Portable/FeaturesResources.resx

```diff
@@ -1451,4 +1451,16 @@ This version used in: {2}</value>
   <data name="Replace_0_with_1" xml:space="preserve">
     <value>Replace '{0}' with '{1}' </value>
   </data>
+  <data name="Introduce_constant" xml:space="preserve">
+    <value>Introduce constant</value>
+  </data>
+  <data name="Introduce_field" xml:space="preserve">
+    <value>Introduce field</value>
+  </data>
+  <data name="Introduce_local" xml:space="preserve">
+    <value>Introduce local</value>
+  </data>
+  <data name="Introduce_query_variable" xml:space="preserve">
```

Comment:
>Introduce_query_variable [](start = 14, length = 24)

Would you mind sharing a screenshot of this scenario? I don't think I've encountered it before.

Reply:
Sure!  Looks like:

![image](https://user-images.githubusercontent.com/4564579/46887828-9ee04b00-ce13-11e8-90c5-05fae650fee2.png)